src/pages/Settings.js

```
@@ -0,0 +1,284 @@
+  // 初回ロード時にカレンダーを取得
+  useEffect(() => {
+    const fetchInitialCalendars = async () => {
+      // カレンダーが既にある場合はスキップ
+      if (localSettings.calendars && localSettings.calendars.length > 0) {
+        setIsInitialLoading(false);
+        return;
+      }
+
+      try {
+        if (window.electronAPI && window.electronAPI.refreshCalendars) {
+          const result = await window.electronAPI.refreshCalendars();
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)